packages/stencil-library/src/utils/utils.spec.ts

```diff
@@ -1,19 +0,0 @@
-import { format } from './utils';
```

Comment:
[DUST] How was `format` being used before, or was it just default boilerplate?

Reply:
Just boilerplate included in Stencil's starter app example!